doc/MANUAL.md

```diff
@@ -136,7 +144,7 @@ If you want the latest and greatest between releases you can grab from GitHub.
 pip install -U git+https://github.com/intel/cve-bin-tool
 ```
 
-CVE Binary Tool relies on a few command line utilities which are usually present
+CVE Binary Tool relies on a few command-line utilities which are usually present
```

Comment:
Convention for "command-line" versus "command line" is a bit more mixed and you see both used in practice, but I'm going to rule no dash here either due to the fact that we use "command line" with a space everywhere else.